backend/sublet/views.py

```diff
@@ -62,11 +71,40 @@ class Properties(viewsets.ModelViewSet):
     """
 
     permission_classes = [SubletOwnerPermission | IsSuperUser]
-    serializer_class = SubletSerializer
+    parser_classes = (MultiPartParser, FormParser, JSONParser)
+
+    def get_serializer_class(self):
+        if self.action == "retrieve":
```

Comment:
maybe condense this to a ternary statement, not opposed to leave it like this though